releasenotes/notes/qpy-serialization-f08c241209706da2.yaml

```diff
@@ -0,0 +1,10 @@
+---
+features_qpy:
+  - |
+    qiskit.qpy now saves and loads circuits reliably using
+    `qiskit.qpy.dump` and `qiskit.qpy.load`. Previously, an
```

Comment:
```suggestion
    The QPY module (:mod:`qiskit.qpy`) now saves and loads circuits with reliably :class:`qiskit.circuit.library.MSGate` gates
    :func:`~qiskit.qpy.dump` and :func:`~qiskit.qpy.dump`. Previously, an
```

Reply:
excellent suggestion, updated to the better language and links